include/dtpcontrols/DataReceptionNode.hpp

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file DataReceptionNode.hpp
+ *
+ * DataReceptionNode is a class derived from 
+ * Its purpose is to control and read status of the data-
+ * reception firmware, within DTP-firmware which is part of DUNE DAQ.
+ *
+ * This is part of the DUNE DAQ Software Suite, copyright 2022.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+
+#ifndef DTPCONTROLS_INCLUDE_DTPCONTROLS_DATARECEPTIONNODE_HPP_
+#define DTPCONTROLS_INCLUDE_DTPCONTROLS_DATARECEPTIONNODE_HPP_
+
+#include "uhal/DerivedNode.hpp"
+
+namespace dunedaq {
+namespace dtpcontrols {
+
+class DataReceptionNode : public uhal::Node {
+
+  UHAL_DERIVEDNODE(DataReceptionNode)
+  
+public:
+  explicit DataReceptionNode(const uhal::Node& );
+  virtual ~DataReceptionNode();
+
+  void reset_input_word_counter(bool);
```

Comment:
I suggest setting a default for the dispatch flag (the name of which should appear in the header, not to leave the meaning of the boolean argument to the imagination of the reader).
I would set the default to `true`, as the actions encoded in methods are typically immediately executed.